### PR TITLE
User2story

### PR DESCRIPTION
**Pull request by LizaMarie0421**

### CODE REVIEW

File: Evaluation/src/main/java/com/revature/controller/QuizController.java [hunk 1]
```diff
@@ -59,4 +59,16 @@ public Quiz findQuizById (@RequestBody Quiz q) {
...
+//	@RequestMapping(value="/quizzes", method = RequestMethod.PUT, consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
+//	@ResponseBody()
+//	public Quiz updateQuizAttempt (@RequestBody Quiz quiz) {
+//		Quiz q1 = qs.findById(quiz.getQuizId());
+//		q1.setQuizAttempt(quiz.getQuizAttempt());
+//		qs.updateQuizAttempt(quiz);
+//		
+//		return q1;
+//	}
+
```
Comment (GattieTime) [inline Evaluation/src/main/java/com/revature/controller/QuizController.java:73]: Why is the committed code commented out?

File: Evaluation/src/main/java/com/revature/controller/UserQuizScoreController.java [hunk 2]
```diff
@@ -7,6 +7,7 @@
 import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.*;
 
+import com.revature.entity.Quiz;
 import com.revature.entity.UserQuizScore;
```
Comment (GattieTime) [inline Evaluation/src/main/java/com/revature/controller/UserQuizScoreController.java:11]: Is this import being used?

File: Evaluation/src/main/java/com/revature/controller/UserQuizScoreController.java [hunk 3]
```diff
@@ -28,5 +29,6 @@
 		
 	}
 	
+
```
Comment (GattieTime) [inline Evaluation/src/main/java/com/revature/controller/UserQuizScoreController.java:33]: Please don't add stand alone blank lines.

File: Evaluation/src/main/java/com/revature/entity/UserQuizScore.java [hunk 4]
```diff
@@ -30,6 +33,10 @@
 	@Column(name="SUBMIT_DATE", nullable=false, columnDefinition="TimeStamp")
 	private Date submitDate;
 	
+
+	
+
+	
 	//We create one transient field for quizId.
```
Comment (GattieTime) [inline Evaluation/src/main/java/com/revature/entity/UserQuizScore.java:40]: Why all these blank new lines?

File: Evaluation/src/main/java/com/revature/entity/UserQuizScore.java [hunk 5]
```diff
@@ -87,11 +115,23 @@ public void setQuizId(Long quizId) {
...
+
+
+
+	
+
+
+
+
+	
 }
```
Comment (GattieTime) [inline Evaluation/src/main/java/com/revature/entity/UserQuizScore.java:137]: please do not add blank lines.

File: Evaluation/src/main/java/com/revature/repo/UserQuizScoreRepository.java [hunk 6]
```diff
@@ -10,4 +10,6 @@
 public interface UserQuizScoreRepository extends JpaRepository<UserQuizScore, Long>, CrudRepository<UserQuizScore, Long> {
 	
 	public List<UserQuizScore> findQuizByUserEmail(String userEmail);
+	
+	
 }
```
Comment (GattieTime) [inline Evaluation/src/main/java/com/revature/repo/UserQuizScoreRepository.java:15]: please do not add blank lines.

File: Evaluation/src/main/java/com/revature/service/QuizService.java [hunk 7]
```diff
@@ -54,5 +54,7 @@ public String deleteQuizById(Long id) {
 		return this.qr.findAll();
 	}
 	
+
+	
 }
```
Comment (GattieTime) [inline Evaluation/src/main/java/com/revature/service/QuizService.java:59]: please do not add blank lines.

File: Evaluation/src/main/java/com/revature/service/UserQuizScoreService.java [hunk 8]
```diff
@@ -41,4 +42,6 @@ public UserQuizScore InsertUserQuizScore(UserQuizScore uqs) {
 		}
 		return quizList;
 	}
+	
+
 }
```
Comment (GattieTime) [inline Evaluation/src/main/java/com/revature/service/UserQuizScoreService.java:47]: please do not just add blank lines.

**Review by GattieTime — CHANGES_REQUESTED** [review]
Please clean up before merge.

File: Evaluation/src/main/java/com/revature/controller/QuizController.java (hunk 1 above)
Comment (LizaMarie0421) [reply Evaluation/src/main/java/com/revature/controller/QuizController.java:73]: We will fix all these thanks!

File: Evaluation/src/main/java/com/revature/controller/QuizController.java [hunk 9]
```diff
@@ -59,4 +59,7 @@ public Quiz findQuizById (@RequestBody Quiz q) {
 	public List<Quiz> getAllQuizzes(){
 		return this.qs.getAllQuizzes();
 	}
+	
+
+
```
Comment (GattieTime) [inline Evaluation/src/main/java/com/revature/controller/QuizController.java:64]: Please remove the blank lines as well.

File: Evaluation/src/main/java/com/revature/service/QuizService.java (hunk 7 above)
Comment (GattieTime) [reply Evaluation/src/main/java/com/revature/service/QuizService.java:59]: Still here

**Review by GattieTime — CHANGES_REQUESTED** [review]

**Review by GattieTime — APPROVED** [review]

**Pull request merged**